### PR TITLE
use next/image

### PR DESCRIPTION
**Pull request by limsohee1002**
### Description:
- Use next/image for images. This will load an image when it is on viewport.
- Fade in when image load.

### Loom recording:
https://www.loom.com/share/dbe6903826cb4006abf466750908c05a

### CODE REVIEW

**Comment by limsohee1002** [comment]
> Are we using [webp](https://www.contentful.com/blog/2019/10/31/webp-source-sets-images-api/) and [sizing our images responsively](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images) when the browser supports it?
> 
> Also, looks like yarn build is failing.

I will fix the yarn build! didn't notice that. 
next/image automatically adds the responsive image but it doesn't actually know what size image needs to be added. I was thinking to do that in separate PR but I will just add that here!:)

**Comment by hhff** [comment]
> next/image automatically adds the responsive image but it doesn't actually know what size image needs to be added. I was thinking to do that in separate PR but I will just add that here!:)

Thank you!

File: utils/flattenImageData.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+export default (imageFile, query) => {
   if (
     typeof get(imageFile, 'url') === 'string' 
   ) {
     return imageFile;
   }
 
+  const imageUrl = get(imageFile, 'fields.file.url', '')
+  const sanitizedQuery = query ? Object.entries(query).map((entry) => `&${entry[0]}=${entry[1]}`).join('') : ''
+  const webpImageUrl = imageUrl.endsWith('.gif') ? imageUrl : `${imageUrl}?fm=webp${sanitizedQuery}`
```
Comment (hhff) [inline utils/flattenImageData.js:12]: what happens if the browser doesn't support webp?

**Comment by limsohee1002** [comment]
@hhff Here is description for the change that I made:

- I remove the `webp` query from the image URL:
next/image actually gets the `webp` image and if it is not supported it will fetch the original format image. 
https://nextjs.org/docs/api-reference/next/image#acceptable-formats here is doc
also, I checked the image is fetched in webp format 
image

- I added sizes value for every image:
https://www.loom.com/share/fc415be7cece434ea0524d1460733412
image is fetched smaller size and when the size gets bigger it fetches the bigger size image.

**Comment by hhff** [comment]
Thank you @limsohee1002 - one last request:
Now that the image fade in, can you give them a really light background color like `#e0e0e0`, so that their container doesn't look empty while the image is loading? :)

**Comment by limsohee1002** [comment]
@hhff now background color is added : https://sanctu-dot-com-hnxnq8mx0-sanctucompu.vercel.app

**Comment by hhff** [comment]
Thank you @limsohee1002 
- [ ] Please update image background to `#f8f8f8`
- [ ] Is it possible to have the image fade in over the image background (rather than replace it?)

**Comment by limsohee1002** [comment]
@hhff I updated the background color and the image was fading in over grey background on my end. I just made the transition a little longer. here is recording: https://www.loom.com/share/2cbd905929e345199db405fab8ead96d
here is preview: https://sanctu-dot-com-abdoknei8-sanctucompu.vercel.app/

**Review by hhff — APPROVED** [review]

**Pull request merged**